braid/_braid.c

```diff
@@ -1962,13 +1974,13 @@ _braid_ComputeFullRNorm(braid_Core  core,
          {
             /* Post send to neighbor processor */
             _braid_CommSendInit(core, level, fi, u, &send_handle);
-            _braid_GridElt(grids[level], send_index)  = -1;
+            _braid_GridElt(grids[level], send_index)  = _braid_SendIndexOff;
             _braid_GridElt(grids[level], send_handle) = send_handle;
          }
          _braid_BaseFree(core, app,  r);
       }
       /* Residual from C-point. */
```

Comment:
@rfalgout Just to make sure that I understand this here -- by using initiali, you will now compute a residual value (possibly nonzero) for the first row of matrix in the periodic case, whereas, for the non-periodic case, we assume the residual is zero.  Right?

Reply:
Correct.  I am not crazy about the name 'initiali' (it was originally 'initial_ci'), but I couldn't think of anything better.  It kind of represents the location of the initial condition in some sense: in the standard case it's at zero, but in the periodic case it is kind of the rightmost point (which is also at index -1 because of the toroidal thing).  Suggestions welcome or we could just keep it as is.  I'd like to keep the name short-ish if possible.

Reply:
Sounds good.  The only other name that came to mind is 
initialcond
but, that could also confuse people potentially.  Perhaps the comment on line 204 in _braid.h, where initiali is introduced, could be expanded a bit to say that this corresponds to the location of the initial condition.

Reply:
I added the word 'condition'  :)